ksql-engine/src/main/java/io/confluent/ksql/services/ServiceContextFactory.java

```diff
@@ -35,15 +36,19 @@ public static ServiceContext create(
         ksqlConfig,
         new DefaultKafkaClientSupplier(),
         new KsqlSchemaRegistryClientFactory(ksqlConfig, Collections.emptyMap())::get,
-        ksqlClient
+        ksqlClient,
+        new DefaultConnectClient(
+            ksqlConfig.getString(KsqlConfig.CONNECT_URL_PROPERTY),
+            Optional.empty())
     );
   }
 
   public static ServiceContext create(
       final KsqlConfig ksqlConfig,
       final KafkaClientSupplier kafkaClientSupplier,
       final Supplier<SchemaRegistryClient> srClientFactory,
-      final SimpleKsqlClient ksqlClient
+      final SimpleKsqlClient ksqlClient,
+      final ConnectClient connectClient
```

Comment:
nit: can we keep the param order here consistent with the order used by `DefaultServiceContext`?  i.e. reverse the order of `ksqlClient` and `connectClient` params